libs/coin-modules/coin-bitcoin/src/psbtFees.ts

```diff
@@ -0,0 +1,87 @@
+import { BigNumber } from "bignumber.js";
+import { PsbtV2 } from "@ledgerhq/hw-app-btc/newops/psbtv2";
+// NOTE: could exist in hw-app-btc and only export the feeFromPsbtBase64 fn from there
+
+/** normalize possible inputs: base64 (with whitespace/URL-safe), or raw hex */
+function normalizeToBuffer(psbtMaybe: string): Buffer | null {
+  if (!psbtMaybe) return null;
+  const s = psbtMaybe.trim();
+
+  // If hex (even length, only [0-9a-fA-F]) → treat as hex
+  if (/^[0-9a-fA-F]+$/.test(s) && s.length % 2 === 0) {
+    try {
+      return Buffer.from(s, "hex");
+    } catch {
+      /* ignore */
+    }
+  }
+
+  // Treat as base64: strip whitespace and convert URL-safe to standard
+  const b64 = s.replace(/\s+/g, "").replace(/-/g, "+").replace(/_/g, "/");
```

Comment:
We can follow the sonarcloud analysis and use replaceAll to make sure we don't forget the /g